src/cumulus/cli.py

```diff
@@ -59,11 +63,24 @@ def main():
 
 
 def start_container(service):
-    call([DOCKER_COMPOSE, "up", service])
+    if service:
+        call([DOCKER_COMPOSE, "up", "-d", service])
```

Comment:
Will this leave us with a way to show high-level output to the user? Do we want the user to see any output?

Reply:
I plan on adding a `cumulus logs <service>` that will run docker-compose logs underneath the hood. As for right now it will just tell them what services are running and how they can access the web app if they have.

Reply:
aight, cool. will we be able to tell anything without that? since if it's running in background, we can't display any messages

Reply:
I'm not sure what you mean here. Alternatively we could make the -d flag optional but once we run this command we have no control over what messages are displayed and what messages aren't displayed.

Reply:
Another issue I see happening is that we won't be able to "easily" display what ports are available to them because there is 2 situations:

1. We don't use -d flag and we execute the `docker-compose up` command, then the cli will be blocked by that command(and might just keep running the cli in the background).

2. We don't use the -d flag but instead we execute `docker-compose up` async, then the cli won't block but we will have to do some of our own magic to see when it is available and what ports are exposed. 

Ultimately, I would prefer to have -d flag be optional to the user, but we need to figure out a solution that will work for that.

Reply:
Right, yea, sorry, I'm dumb. I think you can get it from docker logs probably if we run in background

Reply:
Actually what we could do is just immediately run `docker-compose logs -f` after we up the containers. What do you think?

Reply:
Although that could be misleading if they hit `ctrl-c` to sigkill the containers but it actually just killed the logs.

Reply:
I think I will just print a message saying that logs are available using `cumulus logs [-f]`

Reply:
I don't think you need to print that, can put it in the docs. I think we should just print basic output (like starting [service], done, and like any basic info they need (like maybe "listening on port x"). And then the actual docker logs (and more detailed CLI logs) can be found from `cumulus logs`